python/ray/rllib/examples/cartpole_lstm.py

```diff
@@ -166,11 +166,12 @@ def close(self):
     tune.run_experiments({
         "test": {
             "env": "cartpole_stateless",
-            "run": "PG",
+            "run": "PPO",
             "stop": {
                 "episode_reward_mean": args.stop
             },
             "config": {
+                "num_sgd_iter": 5,
```

Comment:
I think it's actually better to keep PG here because of the lack of complexity; why does a better performance for this test matter? 

I can see why stability would be good but does this test fail often?

Reply:
PPO actually is much faster and more reliable here. Stability is super important since this test runs until the reward is reached, and if PG training collapses that is not good.